receiver/http.go

```diff
@@ -148,6 +168,19 @@ func (htt *HTTP) Start() error {
 		serveMux.Handle(idx, receiver{Handler: h.H, settings: htt, auth: htt.Auth[idx]})
 	}
 
+	if len(htt.AcceptableCAs) > 0 {
+		pool, err := loadClientCertificateCAs(htt.AcceptableCAs)
+		if err != nil {
+			httpLog.WithError(err).Error("Failed to load Client Certificates")
+			return err
+		}
+		server.TLSConfig = &tls.Config{
+			ClientCAs:  pool,
+			ClientAuth: tls.RequireAndVerifyClientCert,
```

Comment:
Should the policy of authentication be configurable? If so, which auth types should we support?

https://golang.org/pkg/crypto/tls/#ClientAuthType

Reply:
See previous comment, but I think "nyeso".

It makes very little sense to accept and/or require Client certificates without verifying them, so the only valid setting I can imagine is disabling it completely (the default) or RequireAndVerifyClientCert.